pages/FAQ.md

```diff
@@ -0,0 +1,38 @@
+# FAQ
```

Comment:
Please remove this header.
As I already outlined on the "How to edit the wiki" page... custom headers like that are pointless, GitHub already adds one themselves, so that is just redundant.